tools/checkcomments/checkcomments.go

```diff
@@ -59,3 +72,51 @@ func run(pass *analysis.Pass) (any, error) {
 
 	return nil, nil
 }
+
+// isIssueopen check the issue open or closed.
+func isIssueOpen(todoText string) bool {
+	var src oauth2.TokenSource
+	if token := os.Getenv("GITHUB_TOKEN"); token != "" {
+		src = oauth2.StaticTokenSource(
+			&oauth2.Token{AccessToken: token},
+		)
+	}
+
+	ctx := context.Background()
+	client := github.NewClient(oauth2.NewClient(ctx, src))
+
+	owner := "FerretDB"
+	repo := "FerretDB"
+	issueNumber, err := getIssueNumber(todoText)
+	if err != nil {
+		log.Fatalf("error in getting issue number: %s", err.Error())
+		return false
+	}
+
+	issue, _, err := client.Issues.Get(ctx, owner, repo, issueNumber)
+	if err != nil {
+		log.Fatalf("error in getting status of issue: %s for issue number: %d", err.Error(), issueNumber)
+		return false
+	}
+
+	return issue.GetState() == "open"
+}
+
+// get the issue number from  issue url.
+func getIssueNumber(todoText string) (int, error) {
+	pattern := `\/issues\/(\d+)`
+	re := regexp.MustCompile(pattern)
```

Comment:
`todoRE` already correctly extracts the issue number only for FerretDB/FerretDB repo. That code would accept issues from any repos. That's not what we want.